### PR TITLE
Some text updates

### PR DESCRIPTION
**Pull request by stubailo**
TODO:

- [ ] If this PR is a new feature, reference an issue where a consensus about the design was reached (not necessary for small changes)
- [ ] Make sure all of the significant new logic is covered by tests
- [ ] Rebase your changes on master so that they can be merged easily
- [ ] Make sure all tests and linter rules pass
- [ ] Update CHANGELOG.md with your change
- [ ] Add your name and email to the AUTHORS file (optional)
- [ ] If this was a change that affects the external API, update the docs and post a link to the PR in the discussion

### CODE REVIEW

File: docs/source/links/state.md [hunk 1]
```diff
@@ -337,9 +353,28 @@ import currentUser from './resolvers/user';
...
-const stateLink = withClientState(
-  merge({}, currentUser, cameraRoll, networkStatus),
-);
+const stateLink = withClientState({
+  cache,
+  resolvers: merge(currentUser, cameraRoll, networkStatus),
+});
+```
+
+You can do the same thing with the `defaults` option as well:
```
Comment (peggyrayzis) [inline docs/source/links/state.md:362]: Nice!!

File: docs/source/links/state.md [hunk 2]
```diff
@@ -414,30 +447,32 @@ write any data. Let's look at an example where we add a todo to a list:
...
+
+        const previous = cache.readQuery({ query });
+        const data = {
+          todos: previous.todos.concat([
+            { id: nextTodoId++, text, completed: false, __typename: 'TodoItem' },
+          ]),
+        };
+
+        // you can also do cache.writeData({ data }) here if you prefer
+        cache.writeQuery({ query, data });
```
Comment (peggyrayzis) [inline docs/source/links/state.md]: shouldn't we return data here so the user has access to the new todo's id?

File: docs/source/links/state.md [hunk 3]
```diff
@@ -469,20 +504,22 @@ can help us toggle one of our todos as completed.
...
+          fragment completeTodo on TodoItem {
+            completed
+          }
+        `;
+        const todo = cache.readFragment({ fragment, id });
+        const data = { ...todo, completed: !todo.completed };
+
+        // you can also do cache.writeData({ data, id }) here if you prefer
+        cache.writeFragment({ fragment, id, data });
+        return data;
```
Comment (peggyrayzis) [inline docs/source/links/state.md]: Why are we returning data here?

**Review by peggyrayzis — CHANGES_REQUESTED** [review]
Just some questions about returning data from mutations we need to address, but otherwise this looks great!! Thanks so much for helping out 🎉

File: docs/source/links/state.md (hunk 2 above)
Comment (stubailo) [reply docs/source/links/state.md]: Oh snap you're right. maybe let's return just the ID and not all the data?

File: docs/source/links/state.md (hunk 3 above)
Comment (stubailo) [reply docs/source/links/state.md]: No idea, let's remove that

**Review by peggyrayzis — APPROVED** [review]

**Pull request merged**